crates/stackable-certs/src/ca/mod.rs

```diff
@@ -66,14 +67,22 @@ pub enum Error {
 
     #[snafu(display("failed to parse AuthorityKeyIdentifier"))]
     ParseAuthorityKeyIdentifier { source: x509_cert::der::Error },
+
+    #[snafu(display(
+        "failed to parse subject alternative DNS name \"{subject_alternative_dns_name}\" as a Ia5 string"
+    ))]
```

Comment:
note: Get rid of the escaped quotes.

```suggestion
    #[snafu(display(
        "failed to parse subject alternative DNS name {subject_alternative_dns_name:?} as a Ia5 string"
    ))]
```